.gitignore

```diff
@@ -8,3 +8,4 @@
 /.idea
 .DS_Store
 .git
+/bin
```

Comment:
is putting binaries under /bin common in go projects? Is this a known convention?

Reply:
I personally prefer to have it under the bin directory, This makes running the built binary easier. i.e ```./bin/promscale```

As I added ```make build``` as ```go build -o bin/promscale ./cmd/promscale``` to avoid git level spam on bin directory added it to `.gitignore` 

But I am open to thoughts/suggestions.

Reply:
The releaser we use puts everything in a dist folder, maybe we should use that so it's unified